api/tests/Behat/bootstrap/v2/Helpers/FixtureHelper.php

```diff
@@ -744,6 +822,45 @@ public function createSuperAdmin(string $testRunId)
         return self::buildAdminUserDetails($this->superAdmin);
     }
 
+    public function createDataForAnalytics(string $testRunId, $timeAgo, $satisfactionScore)
+    {
+        $startDate = new \DateTime($timeAgo);
+
+        $this->createOrgUserClientNamedDeputyAndReport(
+            $testRunId.'_1',
+            User::ROLE_PROF_NAMED,
+            'analytics-prof-submitted',
+            Report::TYPE_104_6,
+            true,
+            true,
+            $startDate,
+            $satisfactionScore
+        );
+
+        $this->createOrgUserClientNamedDeputyAndReport(
+            $testRunId.'_2',
+            User::ROLE_PA_NAMED,
+            'analytics-pa-submitted',
+            Report::TYPE_104_6,
```

Comment:
just noticed this is wrong type. will let someone review first anyway to save a build ;)

Reply:
This been resolved?

Reply:
This one was right. The prof one was wrong. Fixed now ;)